docroot/modules/custom/cs_generated_content/generated_content/node/civictheme_page_variations/02.banner.inc

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/**
+ * @file
+ * Variations for CivicTheme Page nodes.
+ */
+
+use Drupal\cs_generated_content\CsGeneratedContentHelper;
+
+/**
+ * Page, Banner.
+ */
+function _cs_generated_content_create_node_civictheme_page__variations__banner() {
+  /** @var \Drupal\cs_generated_content\CsGeneratedContentHelper $helper */
+  $helper = CsGeneratedContentHelper::getInstance();
+
+  $variations = [
+    // Page, Banner.
+    [
+      'title' => 'Light Theme banner with breadcrumb, with background, without featured image, Content component, Accordion component below banner',
```

Comment:
Just going to note that these long titles and long breadcrumbs - make the generated content shorter - should we shorten titles to say:

"Banner variant 1"
And add the long title to the "summary field"? Or a content component?

Reply:
we need to be able to check these out from the admin UI without going into them

That is why generated content is using 3 first  letters of each prop with a value